apiExamples/selected.html

```diff
@@ -1,10 +1,6 @@
-<auro-input id="matchWordInput" required>
-  <span slot="label">Enter a value to match in the menu</span>
-</auro-input>
-<br />
-<auro-menu id="matchWordMenu">
+<auro-menu>
```

Comment:
Why did you remove the `matchWord` example? The highlight menu option should be an additional example. Removing this example has also caused your build to fail.

Reply:
Hi @jordanjones243,

sorry I accidentally deleted that, i will back the example and add the example new for the highlight menu.